src/components/ParkSign/styles.module.css

```diff
@@ -1,27 +1,30 @@
 .park-sign-wrapper {
-	display: flex;
-	justify-content: center;
-	align-items: center;
-	background-color: var(--tp-primary-blue);
-	padding: 2rem;
-	max-width: 75%;
-	text-align: center;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: var(--tp-primary-blue);
+  padding: 2rem;
+  max-width: 75%;
+  text-align: center;
 
-	min-width: 50%;
-	
-	border: solid white;
-	border-radius: 10px;
-	margin: 2rem 0 2rem 0;
+  min-width: 50%;
+
+  border: solid white;
+  border-radius: 10px;
+  margin: 2rem 0 2rem 0;
 }
 
-.sign-h2 {
-	font-size: 3rem;
+.sign-h1 {
+  font-size: clamp(1.75rem, 1.21rem + 2.38vw, 3rem);
```

Comment:
really cool usage of clamp() totally seems like an easier way than earlier tried methods

Reply:
Agreed with @rsmith90. `clamp` might be how we want to move forward.